docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2121,6 +2121,34 @@ rpc-ws-port="6174"
 The Websockets JSON-RPC listening port (TCP). The default is 8546. You must
 [expose ports appropriately](../../HowTo/Find-and-Connect/Configuring-Ports.md).
 
+### security-module
+
+```bash tab="Syntax"
+--security-module=<NAME>
+```
+
+```bash tab="Command Line"
+--security-module=hsm_plugin
```

Comment:
hsm_plugin? Maybe "crypto_provider". Its not necessarily using a hsm (could just be a local file)

Reply:
Based on the thread I tagged you in Byron maybe change `hsm_plugin` to `security_module` as the example and in the description add a _For example, a Hardware Security Module (HSM) or V3 filestore plugin_

Reply:
Done.